web_modules/Navbar/index.css

```diff
@@ -2,6 +2,20 @@
   display: flex;
   align-items: center;
   justify-content: space-between;
+  height: 50px;
+  background-color: rgba(0, 0, 0, 0.75);
+  position: fixed;
+  width: 100%;
 }
 
-.item { /* placeholder */ }
+:local(.item) {
```

Comment:
You should not have to use `:local` (css modules are on by default-> https://github.com/postcss/postcss.org/blob/28813c775e5500441095867e58204cd7944f4a54/scripts/build.js#L95)